### PR TITLE
Full code reformat

### PR DESCRIPTION
**Pull request by tomerblecher**

### CODE REVIEW

**Review by tomerblecher — COMMENTED** [review]
@AmirSoln All files checked, please approve.

File: lib/pages/error.dart [hunk 1]
```diff
@@ -4,11 +4,19 @@ import 'package:flutter/material.dart';
...
+        Icon(
+          Icons.warning,
+          size: 150,
+          color: Colors.orangeAccent,
+        ),
+        Text(
+          "Well thats Awkward but.. You shoulden't be here.",
+          style: TextStyle(fontSize: 17),
+        ),
         Text("I propebly messed up somewhere, like I always do...")
```
Comment (AmirSoln) [inline lib/pages/error.dart:20]: Typo => probably?

File: lib/services/network/authorization/identity_bloc.dart [hunk 2]
```diff
@@ -7,32 +6,35 @@ import 'package:rxdart/rxdart.dart';
...
+      case 200:
+        {
+          _identitySubject.sink.add(res);
+          // TODO - sink success.
+        }
+        break;
+      case 401:
+        {
+          _identitySubject.sink.addError(NetworkError(
+              res.statusCode, "One of the credentials is incorrect!"));
```
Comment (AmirSoln) [inline lib/services/network/authorization/identity_bloc.dart:24]: Consider using constants. Should be a file /app/src/main/res/values/strings.xml that defines them

File: lib/services/network/http_override.dart [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 /// Override SSL self signed certificate warning.
 ///
-class MyHttpOverrides extends HttpOverrides{
+class MyHttpOverrides extends HttpOverrides {
   @override
-  HttpClient createHttpClient(SecurityContext context){
+  HttpClient createHttpClient(SecurityContext context) {
     return super.createHttpClient(context)
-      ..badCertificateCallback = (X509Certificate cert, String host, int port)=> true;
+      ..badCertificateCallback =
```
Comment (AmirSoln) [inline lib/services/network/http_override.dart:10]: Check the '..' in here

**Review by AmirSoln — CHANGES_REQUESTED** [review]

**Review by AmirSoln — APPROVED** [review]

**Pull request merged**